test/services/fetchers/CryptoCompareHistoHourFetcher.test.ts

```diff
@@ -11,7 +11,7 @@ chai.use(chaiAsPromised);
 
 const {expect} = chai;
 
-describe.only('CryptoCompareHistoHourFetcher', () => {
```

Comment:
Looks like it breaks the CI. Should I add `.only` in my ByBitFetcher.test as well?

Reply:
this "only" should not be here, please remove or you can make is `.skip` if necessary

Reply:
I added skip in the failed ones and this one I keep without `.only`